### PR TITLE
Include OpenDebugAD7 in VS redist

### PR DESCRIPTION
**Pull request by andrewcrawley**
OpenDebugAD7 and DebugEngineHost.VSCode will now be built in all configurations and included in the VS redist NuGet package.

### CODE REVIEW

File: src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef [hunk 1]
```diff
@@ -52,4 +52,28 @@
...
+; "AddressBP"=dword:00000001
+"AlwaysLoadLocal"=dword:00000001
+"AlwaysLoadProgramProviderLocal"=dword:00000001
+; "CallStackBP"=dword:00000001
+; "ConditionalBP"=dword:00000001
+; "DataBP"=dword:00000001
+; "Disassembly"=dword:00000001
+"EnginePriority"=dword:00000050
+; "Exceptions"=dword:00000001
+; "FunctionBP"=dword:00000001
```
Comment (pieandcakes) [inline src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef]: did we want to support FunctionBPs?

File: src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef [hunk 2]
```diff
@@ -52,4 +52,28 @@
...
+"Adapter"="$PackageFolder$\vscode\OpenDebugAD7.exe"
+"Language"="C++"
+"LanguageId"="{3A12D0B7-C26C-11D0-B442-00A0244A1DD2}"
+; "AddressBP"=dword:00000001
+"AlwaysLoadLocal"=dword:00000001
+"AlwaysLoadProgramProviderLocal"=dword:00000001
+; "CallStackBP"=dword:00000001
+; "ConditionalBP"=dword:00000001
+; "DataBP"=dword:00000001
+; "Disassembly"=dword:00000001
```
Comment (pieandcakes) [inline src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef]: Did we want to support Disassembly when in VS?

Comment (andrewcrawley) [reply src/MIDebugEngine/Microsoft.MIDebugEngine.pkgdef]: Eventually, but all the necessary requests would need to be implemented in OpenDebugAD7 first.  Out of scope for now.

**Comment by pieandcakes** [comment]
@andrewcrawley Make sure that you update the redist artifact also. I think thats different than the Nuspec.

**Pull request merged**